ci/release/README.md

```diff
@@ -0,0 +1,140 @@
+# CI CD
+
+## Introduction
+
+- `Semantic release` help us to automates the whole package release workflow including:
+  - Determining the next version number,
+  - Generating the release notes,
+  - Publishing the package.
+- New features and fixes are `immediately` available to users on Staging repository.
+- Use formalized `commit message convention` to document changes in the codebase.
+
+## Credentials
+
+### Github Token
+
+![github_token](img/github_token.png)
+
+Then, this environment variable is ready to use `${{ secrets.GITHUB_TOKEN }}`.
+
+### Public/Private Key
+
+```shell
+$ gpg --full-gen-key
+$ gpg --list-keys
+$ gpg --keyserver keyserver.ubuntu.com --send-keys 193EAE47
+$ gpg --export-secret-keys 193EAE47 | base64
+```
+
+Configure values for:
+```properties
+SIGNING_KEY_ID = 193EAE47
+SIGNING_PASSWORD = password
+SIGNING_KEY = gpg --export-secret-keys 193EAE47 | base64
+```
+
+### Actions Secrets
+
+Create these actions secrets used by release process:
+
+![actions_secrets](img/actions_secrets.png "title")
+
+## Repository Manager
+
+The artifacts are deployed to Sonatype OSSRH (Sonatype Open Source Software Repository Manager).
+
+These are the repositories availables:
+- Snapshots: https://s01.oss.sonatype.org/content/repositories/snapshots
+- Staging: https://s01.oss.sonatype.org/content/repositories/staging
+- Releases: https://s01.oss.sonatype.org/content/repositories/releases
+
+![repositories](img/repositories.png)
+
+## Release Process
+
+- Every push to pre-release branches `beta` deploy an artifact to `snapshots repository` and it's ready to be used.
+- Every push to release branches `main`, `maintenance` deploy an artifact to `staging repository` and it's ready to be used.
+- If there are no restrictions/problems/impediments next step is release the artifact to `maven central`.
+- There is needed a `manual release` to send the libraries to `maven central`.
```

Comment:
What is this manual process?  Do I need to run a script?

Reply:
Currently there is a UI when we could promote an library from `staging` to `release` to maven central

<img width="1013" alt="image" src="https://user-images.githubusercontent.com/4554485/202500048-332b280d-9b58-41dd-bfa2-018d6d2fb768.png">

Another way is to create a Job to deploy version from `staging` to `release` automatically every 5 days for example. Discussion at https://github.com/substrait-io/substrait-java/pull/102#issuecomment-1311059317